src/shipchain/AwsPrivateKeyDBFieldEncryption.ts

```diff
@@ -15,9 +15,9 @@
  */
 
 import { PrivateKeyDBFieldEncryption } from '../entity/encryption/PrivateKeyDBFieldEncryption';
-import { DBFieldEncryption } from '../entity/Wallet';
 import EthCrypto from 'eth-crypto';
 import { getAwsSecret } from './utils';
+import { DBFieldEncryption } from '../entity/encryption/DBFieldEncryption';
```

Comment:
In general there is no strict rule on the placements of imports; however, I prefer if we keep common imports near each other.  Move this line back below the other `from '../entity/...` line above.